### PR TITLE
ML fix front-end bugs

### PR DESCRIPTION
**Pull request by soupette**
#### Description of what you did:
This PR needs to be merged after #5606

### CODE REVIEW

File: examples/getstarted/extensions/upload/config/settings.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-  //   cloud_name: '',
-  //   api_key: '',
-  //   api_secret: '',
-  // },
+  provider: 'cloudinary',
+  providerOptions: {
+    cloud_name: 'cloud-name',
+    api_key: 'api-key',
+    api_secret: 'api-secret',
+  },
```
Comment (HichamELBSI) [inline examples/getstarted/extensions/upload/config/settings.js]: Should we keep this?

File: packages/strapi-plugin-upload/admin/src/containers/HomePage/index.js [hunk 2]
```diff
@@ -204,6 +211,28 @@ const HomePage = () => {
...
+
+    try {
+      await deleteMedia(id);
+
+      dispatch({
+        type: 'ON_DELETE_MEDIA_SUCCEEDED',
+        mediaId: id,
+      });
+    } catch (err) {
+      // Silent
```
Comment (HichamELBSI) [inline packages/strapi-plugin-upload/admin/src/containers/HomePage/index.js]: Should we display a notification here?

File: examples/getstarted/extensions/upload/config/settings.js (hunk 1 above)
Comment (soupette) [reply examples/getstarted/extensions/upload/config/settings.js]: I have commented it but yes we need to

File: packages/strapi-plugin-upload/admin/src/containers/HomePage/index.js (hunk 2 above)
Comment (soupette) [reply packages/strapi-plugin-upload/admin/src/containers/HomePage/index.js]: The notification is already displayed with the `mediaDelete`

File: packages/strapi-plugin-upload/admin/src/utils/formatFileForEditing.js [hunk 3]
```diff
@@ -7,7 +7,8 @@ const formatFileForEditing = file => {
     abortController,
     id: file.id,
     file: {
-      ...pick(file, ['size', 'ext', 'width', 'height', 'url', 'mime', 'name']),
+      ...pick(file, ['size', 'ext', 'width', 'height', 'mime', 'name']),
+      url: get(file, ['formats', 'thumbnail', 'url']),
```
Comment (HichamELBSI) [inline packages/strapi-plugin-upload/admin/src/utils/formatFileForEditing.js]: I'm not sure but is it the URL that is used when the user needs to copy the URL of the file? If yes, should we override the URL here?

Comment (soupette) [reply packages/strapi-plugin-upload/admin/src/utils/formatFileForEditing.js]: Good catch i'll add another info!

Comment (soupette) [reply packages/strapi-plugin-upload/admin/src/utils/formatFileForEditing.js]: In fact here it's best to use the url

**Pull request merged**